files/disable-inactive-freeipa-users.service

```
@@ -0,0 +1,7 @@
+[Unit]
+Description=Disable inactive FreeIPA users.
+
+[Service]
+ExecStart=/usr/local/sbin/disable_inactive_freeipa_users.sh
```

This file should be converted to a template, with the change:
```suggestion
ExecStart=/usr/local/sbin/disable_inactive_freeipa_users.sh "{{ freeipa_days_before_inactive }}"
```

---

See commit [c07572d](https://github.com/cisagov/ansible-role-freeipa-server/pull/73/commits/c07572dc31b7825c298b813d5de7e5edb47afcca).